charmcraft.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+type: "charm"
+bases:
+  - build-on:
+    - name: "ubuntu"
+      channel: "20.04"
+    - name: "ubuntu"
+      channel: "22.04"
```

Comment:
Shouldn't this be dropped?

Reply:
How so? I took the same file content we had in our charms